engines/billing/questions.md

```diff
@@ -0,0 +1,22 @@
+Questions
+====================
+
+1. Should admin be able to see subscriptions?
+
+2. Should user be able to upgrade or downgrade subscriptions?
+
+3. Should confirmations emails be allowed?
```

Comment:
I like the idea of letting braintree send the reminder emails - but it might be a security issue. I don't like the idea of braintree being able to send emails on our behalve. I would prefer it if the emails had a sender that makes clear it's not us. But i guess that might be another option. Would people fail for phishing mails (please enter your password here) from payments@provider.org ?
@elijh what do you think?